src/containers/catalog/CatalogReducer.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
+import * as actionTypes from './CatalogActionTypes';
+
+export const INITIAL_STATE = {
+  asyncState: {
+    isLoading: true,
+    errorMessage: ""
+  },
+  entitySets: []
+};
+
+export function reducer(state = INITIAL_STATE, action) {
+  switch (action.type) {
+    case actionTypes.ENTITY_SET_LIST_REQUEST:
+      return Object.assign({}, state, {
```

Comment:
this looks like `INITIAL_STATE`. does it make sense to just return `INITIAL_STATE` here?